src/Vm.sol

```diff
@@ -2,6 +2,108 @@
 pragma solidity >=0.6.0 <0.9.0;
 pragma experimental ABIEncoderV2;
 
+interface VmSafe {
```

Comment:
Looks like some cheats from master are missing in this file, such as `allowCheatcodes`

Reply:
Ah sorry didn't realize that wasn't done yet, was going to say `makeAddr` is also missing too. Cool let me know once that's ready and I'll take a closer look + ping some people for testing

Reply:
I can do it tomorrow (also open for anyone to take).

Reply:
New cheatcode was added so needs another merge from main—I'll start testing this tomorrow!